docker/.env.example

```
@@ -7,7 +7,14 @@ API_AUTH_REQUIRED=true
 # Public DSN for the Python backend's Sentry project.
 SENTRY_BACKEND_DSN=
 SENTRY_ENVIRONMENT=production
+# Set to 1 to disable Sentry for every Python service.
+DISABLE_SENTRY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISABLE_SENTRY key should go before the SENTRY_BACKEND_DSN key

(UnorderedKey)